src/common/rbac/const.go

```diff
@@ -27,6 +27,8 @@ const (
 	ActionUpdate = Action("update")
 	ActionDelete = Action("delete")
 	ActionList   = Action("list")
+	// test for webhook policy
+	ActionTest = Action("test")
```

Comment:
I don't think we need this action, whoever can update/create a webhook policy can test it.

Reply:
will fix it

Reply:
> I don't think we need this action, whoever can update/create a webhook policy can test it.

will fix it